commands/bootstrap/__tests__/bootstrap-command.test.js

```diff
@@ -166,6 +166,44 @@ describe("BootstrapCommand", () => {
     });
   });
 
+  describe("with --ci", async () => {
+    const originalNpmUserAgent = process.env.npm_config_user_agent;
+
+    afterEach(() => {
+      process.env.npm_config_user_agent = originalNpmUserAgent;
+    });
+
+    it("should call npmInstall with ci subCommand if on npm 5.7.0 or later", async () => {
+      const testDir = await initFixture("basic");
```

Comment:
Due to the requirements of `npm ci`, this means we should indeed create another fixture with the correct `package-lock.json` files (the publish integration tests also use the fixtures used by the unit tests). I would just add one package, have it depend on `tiny-tarball` (simple, no transitive dependencies, very small package-lock), and that should be sufficient.